src/main/scala/is/hail/methods/CalculateConcordance.scala

```diff
@@ -40,53 +40,56 @@ class ConcordanceCombiner extends Serializable {
     }
   }
 
+  def nDiscordant: Long = {
+    var n = 0L
+    for (i <- 2 to 4)
```

Comment:
If I understand correctly, this is just:
```
mapping(2, 3).toLong + mapping(3, 2) + mapping(2, 4) + mapping(4, 2) + mapping (3, 4) + mapping (4, 3)
```
which should be a lot more efficient.